aws-testing/utils_neuron.py

```diff
@@ -494,31 +500,57 @@ def build_grid_space_input_hidden(self, input_dim=2048, hidden_dim=7168, min_seq
             32: 2,
             64: 1,
         }
-        tp_degrees = [4, 16, 64]
-        tp_degrees = [d for d in tp_degrees if min_tp is None or d >= min_tp]
-        tp_degrees = [d for d in tp_degrees if max_tp is None or d <=  max_tp]
+        # All tp,ep tuples
+        tp_ep_combinations = [
+            (1, 16), (1, 64),
+            (4, 1), (4, 16), 
+            (16, 1), 
+            (64, 1), 
+        ]
+        
         kwargs={
             'dtype': dtype,
             'input_dim': int(input_dim),
             'hidden_dim': int(hidden_dim),
         }
-        for tp_degree in tp_degrees:
-            mesh_spec = {"fsdp": -1, "model": tp_degree}
-            batch = batch_sizes[tp_degree]
-            for E in [1, 8, 16, 64, 128, 256]:
+        
+        for tp_degree, ep_degree in tp_ep_combinations:
+            if min_tp is not None and tp_degree < min_tp:
+                continue
```

Comment:
respect min_tp only when ep_degree is 1 for now, we can filter more as needed. min_tp existed because it wouldn't fit the model; but with EP now it can